Functions/src/FunctionsArrayAverage/ArrayAverage.java

```diff
@@ -0,0 +1,39 @@
+package FunctionsArrayAverage;
+
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class ArrayAverage {
+
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Please enter the length of the array:");
+        int x = sc.nextInt();
+        int[] arr = getArray(x);
+        System.out.println("Your array is: " + Arrays.toString(arr));
+        int average = getArrayAverage(arr);
+        System.out.println("The average of the array is: " + average);
+    }
+
+    public static int[] getArray(int x) {
```

Comment:
Functia ar fi putut sa includa si afisarea mesajului de introducere a lungimii array-ului, cat si citirea acesteia. In acest fel, cele doua operatiuni care contribuie la citirea array-ului de la tastatura se fac impreuna,